packages/libsql-client/src/hrana.ts

```diff
@@ -227,12 +227,28 @@ export abstract class HranaTransaction implements Transaction {
     }
 }
 
+export function isHranaPragmaStmt(stmt: hrana.Stmt): boolean {
+    if (typeof stmt.sql === "string") {
+        return stmt.sql.toUpperCase().startsWith("PRAGMA ");
```

Comment:
that's too eager - `pragma foreign_keys` is explicitly documented to not work in transaction context, but there are lots of pragmas that do, and sometimes you may need them to be atomic. Think pragmas that check if the schema is in some particular shape